doc/basic_configuration.rst

```diff
@@ -100,3 +121,12 @@ repository::
 
 You can extract the repository PGP key from the ``buildfarm_deployment``
 configuration.
+
+
+Update URLs to point to required repositories
+---------------------------------------------
+
+During job execution, access to repositories which contain the necessary tools to run the ROS build farm is required.
+These must be specified in your ``ros_buildfarm_config``'s ``index.yaml`` as ``prerequisites``.
+You can use the official ROS repository or mirrors of it.
+See the `Configuration Options <https://github.com/max-krichenbauer/ros_buildfarm/blob/documentation-improvements/doc/configuration_options.rst#entry-point-yaml>`_ documentation page for details.
```

Comment:
This shouldn't link to your fork but be relative within this repo.

Reply:
Oh, sorry - seems like I copy-pasted from the wrong browser tab.
Fixed now.

Reply:
Please use a relative link instead.

Reply:
Done.